src/kirby/components/form-field/textarea/textarea.component.html

```diff
@@ -0,0 +1,2 @@
+<ng-content *ngIf="!value"></ng-content>
```

Comment:
Could reference the `templateRef` of the `contentChild` instead, as this will not synchronize the life cycle hooks between this component and the provided textarea component. With ng-content, the host has no control over the transcluded content (eg, `onDestroy` hook will not trigger on transcluded component when the host gets destroyed - maybe not what we want here.

For more info in this, check this article by the author of Spectator and Akita Netanel Netanel Basal [here, read from "Supporting Lazy Content"](https://netbasal.com/creating-behavioral-components-in-angular-8db6080f0cc4)